### DISCUSSION TITLE
测试文档优化

### DISCUSSION THREAD

**Discussion by bytemain** [opensumi/core#477] [Ideas]
1. 测试进度也要发成 issue， 比如 2.14.4 pre-release electron
这个 issue 的内容最好不要维护在 issue template 中，而是我们自己放在一个文件中，使用 action 或者机器人来创建这两个 issue
或者测试 issue 可以单独发在一个仓库里，在那个仓库里可以设置一些 issue tag

2. 大家以代码贡献的方式维护每一次需要的测试用例。
测试这一套模板最好不基于语雀编辑器了，用 markdown 就好。

**Comment by erha19** [comment]
1. 测试模板可以放到 Wiki 里面，这样也便于大家一起修改，不需要有额外的 commit 记录
2. action/机器人 里面可以基于 pre-release 的插件去 wiki 中拉取两个测试模板创建响应的 Issue，格式如：`TEST: OpenSumi Pre-Release 2.14.4-next` 模板内列有 Electron 和 Web 的测试模板 , 同时添加 `regression testing` 标签，以及根据轮值名单自动 assign 测试人员（issue 中是可以 assign 多人的）
